crate_universe/src/rendering.rs

```diff
@@ -123,6 +139,13 @@ impl Renderer {
         let header = self.engine.render_header()?;
         starlark.push(Starlark::Verbatim(header));
 
+        // Load `transition_opt_alias` if `transition_opt=True`.
+        if self.transition_opt {
+            starlark.push(Starlark::Verbatim(
+                r#"load("//:transition_opt_alias.bzl", alias = "transition_opt_alias")"#.to_owned(),
```

Comment:
Instead of overwriting `alias`, which was the easiest for Starlark serialization, I could allow this to be specified on a per `crate.annotation()` basis and serialize explicitly below `alias()` or `transition_opt_alias()`.  Does that seem worth doing?  As is, to individually opt a crate out means directly referring to the target instead of through the alias.